coc.md

```diff
@@ -0,0 +1,48 @@
+## Code of Conduct
+
+### What is this code of conduct for?
+
+Appium is a piece of technology, but **the core of the Appium community is the people in it**. As such, we are committed to providing a friendly, safe and welcoming environment for all, regardless of gender, gender identity and expression, sexual orientation, ability, physical appearance, body size, race, age, socioeconomic status, religion (or lack thereof), or other marginalized aspect of comunity members. We expect all members of the Appium community to abide by this Code of Conduct whenever interacting in Appium venues (pull requests, GitHub issues, 1-1 or group chat, meetups, conferences, etc...)
+
+### Examples of inappropriate behavior
+
+Because we come from a variety of backgrounds, we don't want to assume that everyone has the same assumptions about what is and isn't appropriate. Here are some examples of inappropriate behavior that are incompatible with our community's ethos:
+
+* Spamming, trolling, intentionally disrupting conversations, or irrelevant solicitation or advertisement
+* Making demeaning or discriminatory comments
+* Making negative assumptions about someone's background, abilities, or intentions
+* Harassing or stalking individuals (online or in person)
+* Giving someone unwelcome sexual attention or making unwelcome physical contact (in the case of an IRL event)
+* Sharing sexual images or using sexually explicit language
+
+In general: treat others how you would like to be treated, were you in their place. Don't be a jerk. _Do_ ask questions. _Do_ keep conflicts productively focused on technical issues. _Do_ think before you speak; remember that what is perceived as a funny witticism in your group of friends might be hurtful or reinforce hurtful stereotypes in the context of our diverse online community. _Do_ remember that we are all people, not robots, and all equally deserving of sensitivity and respect. (If and when robots join our community, let's treat them with respect too!)
+
+
+### What will organizers do about inappropriate behavior?
+
+If we notice you doing or saying something appropriate, an organizer will explain why it's inappropriate and ask you to stop. We won't demonize or vilify you. But please do stop the inappropriate behavior so we can get back to writing and discussing code in a safe environment. If you have philosophical disagreements about what's actually inappropriate, please take them to a separate public or private conversation with an Appium maintainer so we don't turn pull requests into an ethics debate.
```

Comment:
a very tiny typo radically changes the meaning of this sentence.
"If we notice you doing or saying something appropriate"

(should be "inappropriate")